1_fetch/src/get_nwis_data.R

```diff
@@ -24,12 +24,7 @@ nwis_site_info <- function(fileout, site_data){
 }
 
 
-download_nwis_site_data <- function(filepath, parameterCd = '00010', startDate="2014-05-01", endDate="2015-05-01"){
```

Comment:
You can delete the `download_nwis_data()` function since we split this up to download each site individually.